doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -451,7 +451,11 @@ zcbor
 Trusted Firmware-M
 ==================
 
-|no_changes_yet_note|
+The minimal TF-M profile no longer silences TF-M logs by default.
+
+.. note::
+   This can be a breaking change if the UART instance used by TF-M is
+   already in use elsewhere, e.g. in modem trace with a UART backend.
```

Comment:
```suggestion
   This can be a breaking change if the UART instance used by TF-M is
   already in use elsewhere, for example, in modem trace with a UART backend.
```